ingests/ihme_child_mortality.py

```diff
@@ -54,7 +50,9 @@ def main(upload: bool) -> None:
 
 
 def get_location_hierachies():
-
+    """
+    Download the location hierachies spreadsheet so that we can filter out the small sub-national regions and the higher level regions we don't want.
+    """
     location_hierachies = "https://ghdx.healthdata.org/sites/default/files/record-attached-files/IHME_GBD_2019_GBD_LOCATION_HIERARCHY_Y2022M06D29.XLSX"
```

Comment:
Instead of repeating the same URL multiple times, you could define it as a global variable at the beginning of the script: `BASE_URL="https://ghdx.healthdata.org/sites/default/files/record-attached-files/"`